mk/top_of_makefile.mk

```diff
@@ -42,6 +42,10 @@ TARGET_ABI = $(TARGET_SYS)
 TARGET_VENDOR = unknow
```

Comment:
Unrelated to this pull request, but is there typo in _unknow_? Should it be _unknown_?